.github/workflows/scripts/check-codeowners.sh

```diff
@@ -48,17 +48,18 @@ check_code_owner_existence() {
         # Check if it is a known component that is waiting on an owner
         if grep -wq "$module" "$ALLOWLIST"; then
           ((ALLOW_LIST_COMPONENTS=ALLOW_LIST_COMPONENTS+1))
-          echo "\"$module\" not included in CODEOWNERS but in the ALLOWLIST"
+          echo "pass: \"$module\" not included in CODEOWNERS but in the ALLOWLIST"
         else
           ((MISSING_COMPONENTS=MISSING_COMPONENTS+1))
-          echo "\"$module\" not included in CODEOWNERS"
+          echo "FAIL: \"$module\" not included in CODEOWNERS"
         fi
       fi
     fi
   done
   echo "there are $ALLOW_LIST_COMPONENTS components not included in CODEOWNERS but known in the ALLOWLIST"
```

Comment:
Would it be good to check for allowed list components as well and only print them if > 0. For example:

```shell
if [ "$ALLOW_LIST_COMPONENTS" -gt 0 ]; then
  echo "---"
  echo "pass: there are $ALLOW_LIST_COMPONENTS components not included in CODEOWNERS but known in the ALLOWLIST"
fi